fhirzeug/generators/python_pydantic/static_files/tests/test_examples/test_basic.py

```diff
@@ -1,3 +1,5 @@
+import pytest
```

Comment:
I think those tests can be done in the not generated part of the lib, since it just uses the header. Similar how it is done in `test_json_serialization.py`
EDIT: You are testing it in the generated and in the other one. Nice work :)